cmd/openshift-install/main.go

```diff
@@ -62,3 +67,12 @@ func runRootCmd(cmd *cobra.Command, args []string) error {
 	logrus.SetLevel(level)
 	return nil
 }
+
+func awsPermissionsCheck() {
+	key := "OPENSHIFT_INSTALL_PLATFORM"
+	value, ok := os.LookupEnv(key)
```

Comment:
This is only one of the ways folks can select the AWS platform.  I think we need to move this check into something deeper, maybe all the way down to [here][1] (which aligns with @abhinavdahiya's [comment][2] as well, although he may not have wanted it down that deep ;).

[1]: https://github.com/openshift/installer/blob/8096592bd29521e9891fe3f9d763cd330edbb415/pkg/asset/cluster/cluster.go#L92-L103
[2]: https://github.com/openshift/installer/pull/444

Reply:
Ok, I moved this.